src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.CoreRT.cs

```diff
@@ -364,6 +489,9 @@ public object GetOrRegisterObjectForComInstance(IntPtr externalComObject, Create
             if (flags.HasFlag(CreateObjectFlags.Aggregation))
                 throw new NotImplementedException();
 
+            if (flags.HasFlag(CreateObjectFlags.TrackerObject))
```

Comment:
Just to make sure that I understand this right: This blocks the newly added code from getting executed for now. Is that correct?

Reply:
I implement only CCW. I thought that throw that for RCW was a right way, but if I remove that lines, then app from #1453 is basically working (except it does not produce any Toast for me). But exit code == 0. So I would remove these lines, and let it leak. At least you wasn't agains that previously.